tests/integration/operator-integ-suite-ocp.sh

```diff
@@ -24,14 +24,47 @@ set -u
 # Print commands
 set -x
 
-# deploy operator in OCP
-echo "--------------------------------"
-echo "Check that istio operator is running in OCP"
-echo "--------------------------------"
-"${WD}"/check-operator.sh
-
-# create a Istio Helm Install CP and test httpbin
-echo "--------------------------------"
-echo "Create a IstioHelmInstall Control Plane and test httpbin"
-echo "--------------------------------"
-"${WD}"/control-plane-test.sh
+OPERATOR_NS="${OPERATOR_NS:-istio-operator}"
+OPERATOR_NAME="${OPERATOR_NAME:-istio-operator}"
+CONTROL_PLANE_NS="${CONTROL_PLANE_NS:-default}"
```

Comment:
Using `default` might hide some things. For instance, we don't need to apply the anyuid or privileged SCC's in the default namespace as it is not that strict.

Suggestion: Use anything else (e.g. istio-system).

Reply:
(and make sure to run the anyuid/privileged workaround)

Reply:
Change made.